plugin/evm/gossiper.go

```diff
@@ -259,7 +270,24 @@ func (n *pushGossiper) awaitEthTxGossip() {
 						"len(txs)", attempted,
 						"err", err,
 					)
+					continue
+				}
+
+				gossipTxs := make([]*GossipEthTx, 0, len(txs))
+				for _, tx := range txs {
+					gossipTxs = append(gossipTxs, &GossipEthTx{Tx: tx})
+				}
+
+				n.ethTxGossiper.Add(gossipTxs...)
+				if err := n.ethTxGossiper.Gossip(context.TODO()); err != nil {
```

Comment:
No sure if this is a valid concern - but it's possible that we added more transactions than we sent here. Could this ever result in a memory buildup?

Reply:
Hmm I guess this can happen if txs are coming in at a rate quicker than they're being gossiped.... but I feel like this is not too concerning since currently we're pretty much running a gossip cycle every time we see a tx I don't really expect this the txs waiting to be gossiped to build up to a noticeable size

Reply:
We could add a call to `Gossip` inside of the `gossipTicker` call if we wanted to further protect against this.

Reply:
I'll add this in.